tools/mkmapdata/mkmapdata.sh

```diff
@@ -263,13 +263,13 @@ fi
 
 if [ "$phys" = "clm4_5" ]; then
     grids=(                    \
-           "0.5x0.5_nomask"     \
-           "0.25x0.25_nomask"   \
-           "5x5min_nomask"     \
-           "10x10min_nomask"   \
-           "0.9x1.25_nomask" \
+#          "0.5x0.5_nomask"     \
+#          "0.25x0.25_nomask"   \
+#          "5x5min_nomask"     \
+#          "10x10min_nomask"   \
+#          "0.9x1.25_nomask" \
            "3x3min_nomask" \
-           "1km-merge-10min_HYDRO1K-merge-nomask" \
+#          "1km-merge-10min_HYDRO1K-merge-nomask" \
```

Comment:
I pushed this change to allow others the option of reproducing the error that I documented [here](https://github.com/ESCOMP/CTSM/pull/823#issuecomment-549087078).

Reply:
Note to self: 
Reverse this change when done generating 3x3min_nomask map files :-)